components/form/UploadImageInput.tsx

```diff
@@ -26,10 +28,11 @@ const UploadImageInput: React.FC<UploadImageProps> = ({
   setError,
   isRequired: isRequired = false,
   imageType,
+  imageSource,
 }) => {
   const supabase = newClient();
   const [userId, setUserId] = useState('');
-  const [isImageUploaded, setIsImageUploaded] = useState(false);
```

Comment:
I think `isImageUploaded` was actually tracking if there was a valid image submitted before we allowed the user to move on so you might need it to fix the regression that's just been introduced